.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
This is just something I like to do, but it's nice to run the specs in a random order so you can spot easily flaky specs.

```suggestion
--require spec_helper
--order random
```

Reply:
Thank you for your suggestion :)